src/assets/scss/cards.scss

```diff
@@ -0,0 +1,115 @@
+@import './style.scss';
```

Comment:
This file is not properly formatted. Please use VSCode built-in to format CSS files. (Right-click on file and select format)

Reply:
Theres no such option available for me

![imagem_2022-04-13_163448978](https://user-images.githubusercontent.com/35656220/163256569-905d4401-e066-4421-a441-dc90d6c35344.png)